_examples/complete/config/istio/virtual-service.yaml

```diff
@@ -7,16 +7,16 @@ kind: VirtualService
 metadata:
   annotations:
     meta.helm.sh/release-name: appname
-    meta.helm.sh/release-namespace: istio-system
+    meta.helm.sh/release-namespace: default
   generation: 1
   labels:
     app.kubernetes.io/instance: appname
     app.kubernetes.io/name: appname
   name: appname
-  namespace: istio-system
+  namespace: default
```

Comment:
why default?

Reply:
its an example of namespace, user can use their namespace name in place of default, there is no depenncies of virtual-service with istio-ingress addon.